Corona-Warn-App/src/main/res/values-v27/styles.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+
+    <style name="AppTheme" parent="BaseTheme">
+        <item name="android:navigationBarColor">@color/colorSystemUI</item>
+        <item name="android:windowLightNavigationBar">@bool/lightSystemUI</item>
+        <item name="android:windowLayoutInDisplayCutoutMode">shortEdges</item>
+    </style>
```

Comment:
What feature here requires API27?

Reply:
This to avoid jumping on devices that have a `cutout` when going to immersive mode